migration/migration.go

```diff
@@ -770,10 +770,10 @@ func createOrUpdateSystemPlannerItemType(ctx context.Context, witr *workitem.Gor
 		workitem.SystemDescription:  {Type: workitem.SimpleType{Kind: "markup"}, Required: false, Label: "Description", Description: "A descriptive text of the work item"},
 		workitem.SystemCreator:      {Type: workitem.SimpleType{Kind: "user"}, Required: true, Label: "Creator", Description: "The user that created the work item"},
 		workitem.SystemRemoteItemID: {Type: workitem.SimpleType{Kind: "string"}, Required: false, Label: "Remote item", Description: "The ID of the remote work item"},
```

Comment:
Shouldn't `workitem.SystemRemoteItemID` be read only as well? 
ID should not change.

Reply:
Maybe, so far I've added it wherever we `continue`d here: https://github.com/fabric8-services/fabric8-wit/pull/1996/files#diff-55f2da878ed2fcb42e5373010f02ff92L500

Reply:
@kwk ok